File: avancado-intermediario/Python/Q5/arvore.py
from objeto import Objeto

class Arvore(Objeto):

    def __init__(self, altura, x = 0.0, y = 0.0):
        super().__init__(x, y)
        self.__altura = altura

    def getAltura(self):
        return self.__altura

    def __str__(self):
        return (f"{super().__str__()}"
                f"Altura = {self.getAltura()}")

File: avancado-intermediario/Java/Q5/Arvore.java
public class Arvore extends Objeto {

    private double altura;

    public Arvore(double altura) {
        this(altura, 0.0, 0.0);
    }

    public Arvore(double altura, double x, double y) {
        super(x, y);
        this.altura = altura;
    }

    public double getAltura() {
        return altura;
    }

    @Override
    public String toString() {
        return super.toString() +
               "Altura = " + altura;
    }
}

